messages/en.json

```
@@ -30,5 +30,6 @@
   "admin/tiktok.disconnect.confirm.button": "Confirm",
   "admin/tiktok.disconnect.confirmation-toast.success": "Account disconnected successfully.",
   "admin/tiktok.disconnect.confirmation-toast.failure": "It was not possible to disconnect the account.",
-  "admin/tiktok.confirmation-modal.title": "Check the information before submitting"
+  "admin/tiktok.confirmation-modal.title": "Check the information before submitting",
+  "admin/tiktok.connection-error.access-token": "You was disconnected because of an AccessToken error."
```

I'm not sure but i think that it should be:

```suggestion
  "admin/tiktok.connection-error.access-token": "You were disconnected because of an AccessToken error."
```